recipes/install.rb

```diff
@@ -267,6 +267,13 @@
               'maximumpoolsize' => 400,
               'taskqueuecapacity' => 20000,
               'description' => 'Hopsworks Conda Executor Service'
+          },
+          'concurrent/jupyterExecutorService' => {
+              'threadpriority' => 8,
+              'corepoolsize' => 300,
+              'maximumpoolsize' => 300,
+              'taskqueuecapacity' => 1000,
```

Comment:
I think we would also benefit from making these parameters configurable.

Reply:
Also I thought we didn't want to set the queue for this pool since it doesn't make much sense to wait

Reply:
I will make the params configurable.
You need the queue to pause connections from tornado that do not have resources to run. If I put no queue, the connection will fail with an error, but tornado(jupyter) will eagerly try to reconnect... and we end up with reconnects every 100ms.